tasks/15_confirm-ending/confirm-ending.js

```diff
@@ -0,0 +1,7 @@
+function confirmEnding(str, target) {
+  return (str.indexOf(target, str.length - target.length) === -1)
```

Comment:
Doesn't this expression returns `boolean` itself? Why you're using ternary operator here?